api/golang/core/lib/enclaves/enclave_context.go

```diff
@@ -96,9 +97,10 @@ func (enclaveCtx *EnclaveContext) RunStarlarkScript(ctx context.Context, seriali
 func (enclaveCtx *EnclaveContext) RunStarlarkScriptBlocking(ctx context.Context, serializedScript string, serializedParams string, dryRun bool, parallelism int32) (*StarlarkRunResult, error) {
```

Comment:
can you add this to RunRemotesStarlarkScriptBlocking?

Reply:
Good catch, done